corehq/apps/app_manager/models.py

```diff
@@ -3399,7 +3399,20 @@ def _column_to_series(column):
                             xpath=suite_xml.Xpath(function='description'))
                     ),
                 ),
-            ] + list(_get_graph_fields())
+            ] + list(_get_graph_fields()) + [
+                suite_xml.Field(
+                    header=suite_xml.Header(
+                        text=suite_xml.Text(
+                            locale=suite_xml.Locale(id=id_strings.report_last_sync())
+                        )
+                    ),
+                    template=suite_xml.Template(
+                        text=suite_xml.Text(
+                            xpath=suite_xml.Xpath(function="instance('reports')/reports/@last_sync")
```

Comment:
Offlined with clayton. @nickpell 

The right thing to do here is send down the last_sync datetime in UTC ISO-8601 format (so like the normal YYYY-mm-ddThh:MM:ssZ), and then wrap the xpath you have here in date and format-date: `"format-date(date(instance('reports')/reports/@last_sync))"`. This is better than trying to format it for the user on the HQ side and displaying what's in the fixture directly, because CommCare uses (or theoretically could use) the user's locale to localize the datetime correctly for the user.